src/compute/src/sink/persist_sink.rs

```diff
@@ -426,6 +408,25 @@ where
         // do this, we would be stuck at `[minimum]`.
         let mut emitted_persist_frontier: Option<Antichain<_>> = None;
 
+        // We go straight to the write handle to learn about the current upper
+        // and upper advancements. The "persist_oks" stream is lying to us! For
+        // example, when starting the persist_source with an as_of, the upper
+        // will jump to that as_of, and I believe REFRESH EVERY might also be
+        // playing tricks on us.
```

Comment:
I don't think `REFRESH EVERY` does anything unexpected here, except that the target shard might be initialized with a `since` that's in the future.

Reply:
yep, will remove that part of the comment!